save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/modal/LogoutModal.kt

```diff
@@ -29,7 +32,7 @@ fun logoutModal(
         )
         if (replyToLogout.ok) {
             // logout went good, need either to reload page or to setUserInfo(null) and use redirection like `window.location.href = window.location.origin`
-            window.location.href = "${window.location.origin}/"
+            navigate("/")
             window.location.reload()
```

Comment:
maybe set `setUserInfo(null)` ?

Reply:
Do not want to put UserInfo also to topbar... Let it be only in two places, where we modify user